networking/networking-dashboards.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="networking-dashboards_{context}"]
+= Networking dashboards
+include::_attributes/common-attributes.adoc[]
+:context: networking-dashboards
+
+toc::[]
+
+Networking metrics are viewable in dashboards within the {product-title} web console by selecting *Observe* -> *Dashboards*.
+
+[id="network-observability-operator-operator-dashboards"]
+== Network Observability Operator
+If you have the Network Observability Operator installed, you can view network traffic metrics dashboards by selecting the *Netobserv* dashboard from the *Dashboards* drop-down list. For more information about metrics available in this *Dashboard* view,
+// 2/9 Network Observability GAs so we will put this back in then.
+//see xref:../network_observability/metrics-alerts-dashboards#network-observability-viewing-metrics-dashboards_metrics-dashboards-alerts[Network Observability metrics dashboards]
+
+[id="general-networking-ovnk-dashboards"]
+== Networking and OVN-Kubernetes dashboard
+You can view both general networking metrics as well as OVN-Kubernetes metrics from the dashboard.
+
+To view general networking metrics, select *Networking/Linux Subsystem Stats* from the *Dashboards* drop-down list. You can view the following general metrics from the dashboard: *Network Utilisation*, *Network Saturation*, and *Network Errors*.
```

Comment:
`You can view the following general metrics` can be changed to `You can view the following networking metrics`